tests/src/Nightwatch/Commands/thunderFillCKEditor.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @file
+ * Sets tag for element. It can be existing tag or new one.
+ *
+ * This provides a custom command, .thunderFillCKEditor()
+ *
+ * @param {string} selector
+ *   The element selector.
+ * @param {string} value
+ *   The HTML value that will be filled to CKEditor.
+ *
+ * @return {object}
+ *   The 'browser' object.
+ */
+
+/* eslint-disable func-names */
+exports.command = function thunderFillCKEditor(selector, value) {
```

Comment:
Not sure about the thunder prefix here

Reply:
I'll remove it. I have prefixed it because I didn't want to have potential command collisions with core commands if they are introduced one day. But, if that happens, we will fix it.